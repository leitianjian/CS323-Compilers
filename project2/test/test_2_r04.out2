Error type 4 at Line 7: compare function is redefined
Error type 8 at Line 15: function return type is not matched
