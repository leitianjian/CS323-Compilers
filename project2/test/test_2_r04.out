Error type 4 at line 7: compare function is redefined
